docs/crocks/Pred.md

```diff
@@ -2,12 +2,65 @@
 
 `Pred a`
 
---
+The `Pred` crock allows you to reuse the same predicate logic with different kind of inputs, the only think we ask of you is that you let `Pred` know how to get from your expected input to the predicate input.
+
+Say you have a predicate function `isOfAge = age => age >= 21`, but you have an input `{name: "John", age : 23}`, you could of course do something like `isPersonOfAge = person => isOfAge(person.age)`
+
+Or you can use `Pred` and gain a monoidal interface, with the ability to decorate your predicate to work with any kind of input, using the contramap interface.
+
+That means that you would have the ability to easily chain your predicates without too much rewrite.
+
+Let's say that you also want to require that a person was referred by a friend, you could just concat 2 preds or use another mechanism for conjunction.
 
---
 
 ```js
---
+// Helpers -- predicate functions
+const person =
```

Comment:
Would prefer (2) space indentation. and a line break between declarations. Like this:

```js
const person =
  {name: "John", age : 23, referral: "ANON"}

const isOfAge =
  age => age >= 21

const isReferred =
  who => who === "FRIEND"
```